python/modem-firmware-1.3+/nrf_cloud_provision.py

```diff
@@ -31,7 +31,17 @@ class ProvisionResult(Enum):
 
 from modem_credentials_parser import write_file
 
-API_URL = "https://api.nrfcloud.com/v1/"
+DEV_STAGE_DICT = {'dev':     '.dev.',
```

Comment:
I just occurred to me that if we are going to add dev and beta support here, we might as well add ca certs for them too.  See my branch: https://github.com/plskeggs/utils/tree/feature-coap-all-stages

Reply:
they use a different CA for dev and beta?
what is the reason for that?

Reply:
IRIS-6923

Reply:
I did not ask.